NextcloudTalk/LoginViewController.m

```diff
@@ -219,6 +244,70 @@ - (void)presentAuthenticationViewWithServerURL:(NSString *)serverURL withUser:(N
     [self presentViewController:navController animated:YES completion:nil];
 }
 
+#pragma mark - Files app accounts
+
+- (void)checkForFilesAppAccounts
+{
+    if (![NCUtils isNextcloudAppInstalled]) {
+        self.importButton.hidden = YES;
+        return;
+    }
+
+    NSString *path = [[[[NSFileManager defaultManager] containerURLForSecurityApplicationGroupIdentifier:appsGroupIdentifier] URLByAppendingPathComponent:kNextcloudFilesDatabaseFolder] path];
+    NSURL *accountsFileURL = [[NSURL fileURLWithPath:path] URLByAppendingPathComponent:kTalkAccountsFileName];
+    NSArray *filesAccounts = [[NKCommon shared] readDataAccountFileAt:accountsFileURL];
+    NSArray *talkAccounts = [[NCDatabaseManager sharedInstance] allAccounts];
+
+    _importedFilesAccount = [NSMutableArray new];
+    for (NKDataAccountFile *filesAccount in filesAccounts) {
+        BOOL accountIncluded = NO;
+        for (TalkAccount *talkAccount in talkAccounts) {
+            if ([talkAccount.server isEqualToString:filesAccount.url] && [talkAccount.user isEqualToString:filesAccount.user]) {
```

Comment:
This will compare case sensitive. Should be done case insensitive here to be sure?